third-party/unittest/CMakeLists.txt

```diff
@@ -77,18 +77,18 @@ endif ()
 target_include_directories(llvm_gtest
   PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/googletest/include>
          $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/googlemock/include>
-         $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/llvm-gtest/>
-         $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/llvm-gmock/>
+         $<INSTALL_INTERFACE:include/llvm-gtest/>
+         $<INSTALL_INTERFACE:include/llvm-gmock/>
```

Comment:
This is undoing a change made recently in https://github.com/llvm/llvm-project/commit/91b3ca39667b6341a8c1983a1467fae14b58318b. Is that intentional or was your checkout just stale?

Reply:
My checkout just stales. Thanks for pointing it out.